arenadata/Dockerfile.ubuntu

```diff
@@ -42,14 +43,19 @@ ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
 ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
 
 # Compile with running mocking tests
-RUN bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash
+RUN set -eux; \
+    export PYTHON=/usr/bin/python2; \
+    bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash; \
+    make -C gpdb_src/src/pl/plpython clean; \
+    export PYTHON=/opt/adb6-python3.9/bin/python3; \
+    bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash
```

Comment:
In this case unit tests will be run twice (for each exec `compile_gpdb.bash`). You should use ENV `SKIP_UNITTESTS=1`.